src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -882,13 +882,25 @@ private static void validatePropertySectionPropertyTokens(String fileName, Strin
         assertEquals("tokens to check", columns.get(1).getTextContent(),
                 fileName + " section '" + sectionName
                         + "' should have the basic token description");
-        assertEquals("subset of tokens " + CheckUtil.getTokenText(check.getAcceptableTokens(),
-                check.getRequiredTokens()),
-                columns.get(2).getTextContent().replaceAll("\\s+", " ").trim(),
+
+        String acceptableTokenText = columns.get(2).getTextContent().trim();
+        final int removePrefixIdx = "subset of tokens".length();
+        acceptableTokenText = acceptableTokenText.substring(removePrefixIdx);
+        assertFalse(isInvalidTokenPunctuation(acceptableTokenText),
+                ", & . should be placed at beginning of the next corresponding lines "
```

Comment:
Put single quotes around `,` and `.` and write out and so it is clear.

Reply:
Done